.github/workflows/tests.yml

```diff
@@ -14,12 +14,22 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - {name: Windows, python: '3.12', os: windows-latest, tox: fail_fast_test_main }
+          - {
```

Comment:
is this accidental or did you really change something?

Reply:
It's my prettier setup that thinks this should be formatted differently :/